rationale/fees.md

```diff
@@ -0,0 +1,36 @@
+Fees
+===
+
+- [Preamble](#preamble)
+- [Background: EIP-1559](#background-eip-1559)
+- [Fee Burning](#fee-burning)
+
+## Preamble
+
+Two desiderata are elastic block size limits—which can smooth out transaction fees and prevent fee spikes—and variable burning of fees—which can respond to market forces and provides intrinsic demand for the burned coin that cannot be replicated by issuance of new supply. Interestingly, these two properties cannot be accomplished independently. Without a negative feedback mechanism that cannot be bypassed, block producers will always produce maximum-size blocks. This negative feedback mechanism is in-protocol variable fee burning. In a similar vein, variable (rather than fixed) fee burning requires a measure of block usage, which happens to exactly be an elastic block size limit.
+
+This document describes the rationale and structure of an elastic block size limit through variable in-protocol fee burning.
+
+## Background: EIP-1559
```

Comment:
Shouldn't this section mention if we do anything different from EIP-1559? I was not familiar with that eip and it's not obvious if there is anything we do really differently here (besides renaming).

Reply:
The only thing that's different here from EIP-1559 is

> Since blocks grow in size by a power of 4 due to the arrangement of data in a square, a more reasonable initial choice of parameters would be 1:4 target:maximum block size limits.

which is mentioned further down. Is that sufficient?

Reply:
Not sure. Yes, from a spec-only perspective, it is sufficient. From an accessibility pov, it would help someone familiar with eip-1559 to immediately grasp how we do fee burning to have a sentence (somewhere) stating that besides that minor difference they are essentially the same.

Reply:
Clarified further in 2e01d72.